### PR TITLE
fix #4537, fix #4538: flask.request.data/form to self.content/form

### LINKED ISSUE #4537: flask.request.data to self.content
_Originally posted by @robnagler in https://github.com/radiasoft/sirepo/issues/4388#issuecomment-1126771252_

### LINKED ISSUE #4538: flask.request.form to self.form
_Originally posted by @robnagler in https://github.com/radiasoft/sirepo/issues/4388#issuecomment-1126771252_

### PR DESCRIPTION
**Pull request by rorour**

### CODE REVIEW

**Review by mkeilman — COMMENTED** [review]
One little style quibble

File: sirepo/importer.py [hunk 1]
```diff
@@ -16,20 +16,20 @@
...
-        form (flask.request.Form): what to import
+        sapi : sapi.form is what to import
 
     Returns:
         dict: data
     """
     from sirepo import simulation_db
 
-    if not 'zip' in form:
+    if not 'zip' in sapi.form:
```
[the remark this answers is not in the document]
Comment (robnagler) [reply sirepo/importer.py:30]: Yes, good catch, and btw, "not in" and "not x in" are not the same operators. While Python does define it as [inverse truth](https://docs.python.org/3/reference/expressions.html#not-in) of "in", it is a different function call to objects, and it's likely people have abused it...

[the remark this answers is not in the document]
Comment (rorour) [reply sirepo/importer.py]: a442a93635e4d7e9b02e7421439aaffa86b25bd9

**Comment by robnagler** [comment]
@rorour I think we should close this and open a new one once you are done with integrating api.Call (to be written) and api.Spec. It will be a different approach that this PR. Agreed?

**Review by mkeilman — APPROVED** [review]
👍

**Comment by rorour** [comment]
Sounds good

**Pull request closed without merging**